.gitignore

```diff
@@ -39,3 +39,4 @@ api.html
 
 brakeman-output.*
 .idea
+TEMP_NOTES.txt
```

Comment:
Do you mean to include this in the final outcome?

Reply:
yeah, it's just a file for my personal notes that i want ignored.